linkml_model/model/schema/meta.yaml

```diff
@@ -2541,6 +2536,145 @@ slots:
       - in the context of RDF, this should be used for slots that can be modeled using the RDF reification vocabulary
       - in the context of property graphs, this should be used on edge classes to indicate which slots represent the input and output nodes
 
+  issued:
+    description: Date of the first vocabulary publication.
+    domain: schema_definition
+    range: date
+    inherited: true
+    slot_uri: dcterms:issued
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - dcterms:issued
+    see_also:
+      - https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/issued
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  modified:
+    description: Last vocabulary modification date.
+    domain: schema_definition
+    range: date
+    inherited: true
+    slot_uri: dcterms:modified
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - dcterms:modified
+    see_also:
+      - https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/modified
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  rights:
+    description: Property rights of the vocabulary.
+    domain: schema_definition
+    range: string
+    inherited: true
+    slot_uri: dcterms:rights
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - dcterms:rights
+    see_also:
+      - https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/rights
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  licence:
+    description: >-
+      Licence of the vocabulary. 
+      Recommended practice is to refer to a rights statement with a URI. 
+      If this is not possible or feasible, a literal value (name, label, or short text) may be provided.
+    domain: schema_definition
+    range: string
```

Comment:
again, it would really help if we could get people that go all the way to developing a LinkML schema to use URIs for licenses, contributors and creators etc.